### PR TITLE
Improve UI consistency

### PR DESCRIPTION
**Pull request by alexanderstephan**
### Motivation and Context
- Unify look
- Introduce more common CSS styles
- Fix some typos

### Description

- Add default styling for text areas and preset images (preset images are used in lecture hall settings and on the watch page)
- Don't render user notifications if there are none
- Improve alignment of course settings
- Theme dropdown menu for selecting a lecture hall
- Use textarea in lecture management card
- Lecture hall settings padding
- Use form container for
  - Audits
  - User notifications
  - Workers

### Steps for Testing

Inspect the UI of all changed pages.

### CODE REVIEW

**Comment by MatthiasReumann** [comment]
FYI: I updated the _Edit-Info-Page_ in #836 according to the changes you made in this PR - so you don't have to ^^. 
Also: I also utilized the `.btn` class.

File: web/template/admin/admin_tabs/workers.gohtml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {{define "workers"}}
-    <h2 class="text-2xl text-1 my-auto mt-4 font-medium">Worker Overview</h2>
-    <div class="dark:border-secondary bg-gray-100 rounded-md overflow-x-auto mb-6 my-3 shadow-md dark:bg-secondary-lighter">
-        <table class="table-auto w-full rounded-sm">
+    <div class="form-container">
+        <table class="table-auto w-full form-container-title">
```
Comment (MatthiasReumann) [inline web/template/admin/admin_tabs/workers.gohtml]: I think this broke the table a little bit (See Screenshot)

Screenshot 2023-04-03 at 13 02 25

**Review by MatthiasReumann — CHANGES_REQUESTED** [review]
Looks absolutely gorgeous! 💯

I left one comment regarding the worker-table but after fixing that this PR is ready to merge.

File: web/template/admin/admin_tabs/workers.gohtml (hunk 1 above)
Comment (alexanderstephan) [reply web/template/admin/admin_tabs/workers.gohtml]: Good catch, probably overlooked something during merging.

**Review by MatthiasReumann — APPROVED** [review]
Thanks!

**Review by joschahenningsen — APPROVED** [review]

**Pull request merged**